ccm_web/server/src/main.ts

```diff
@@ -47,6 +48,11 @@ async function bootstrap (): Promise<void> {
   const sessionStore = new SequelizeStore({ db: sequelize, tableName: 'session' })
   sessionStore.sync({ logging: (sql) => logger.info(sql) })
 
+  // this is to increase the size limit for request payload, which defaults to 100kb
+  // then the array element limit will be enforced by the class-validator in CreateSectionsDto
+  app.use(json({ limit: '50mb' }));
+  app.use(urlencoded({ extended: true, limit: '50mb' }));
```

Comment:
Is `50mb` a good size for the amount of data to be uploaded?  It seems too large.  Is it based on the total size of 60 lines of data with each field filled to its maximum capacity?  In this case, 60 lines of section IDs, which is just a few bytes each.

It would be better to specify the max. number of lines in some variable, then calculate a reasonable file size for that here.

The max. file size (in kb or mb) and the max. number of sections should be shown in the UI as well, so if the file size error or the number of lines exceeded error comes up, the user will not be caught by surprise.

What kind of error is generated when a file too large or too many lines is uploaded?  Is the error caught and shown to the user in an easily understandable way?

Reply:
> Is 50mb a good size for the amount of data to be uploaded? 
I am not sure what is the best number for the limit here. I was testing with the file [sections2.csv](https://github.com/tl-its-umich-edu/canvas-course-manager-next/files/7313298/sections2.csv) as mentioned in the #217 description, which has size of 1.7MB. 

I agree that ideally the upload file size should not be too big, and should be within the limit of kb, less than MB.

But I figure once we setup up the payload limit to a big enough number, the upload process won't error out at this stage, but rather pass through all data into the next check for section array size ( less than 60 is allowed), and then user will get a nicely formatted message in UI.

> This does seem to be the way to do this, although the Nest folks seem to recommend using body-parser..
I saw the body-parser suggestion and also the suggest as in this PR. 

> My question is if this is really needed if we never hit the default limit with the line limits in place...
I did that since that was the error originally reported in #217

Reply:
Okay, you are right, something like this is needed, since the record numbers aren't considered until the body is parsed.

I think maybe `50mb` is excessive, no one is going to upload something that size, and if they did, I wouldn't want the application to hang while processing it. How about something like `5mb`? Remember only `.csv` files are accepted.

I will test this this afternoon.

Reply:
It is 5mb now